dashboard/lib/contact_rollups_v2.rb

```diff
@@ -1,8 +1,8 @@
 class ContactRollupsV2
   def self.build_contact_rollups(log_collector)
-    log_collector.time!('Truncate tables') do
-      ContactRollupsRaw.truncate_table
-      ContactRollupsProcessed.truncate_table
+    log_collector.time!('Deletes intermediate content from previous runs') do
+      ContactRollupsRaw.delete_all
```

Comment:
I have to use `delete_all` instead of `truncate`  so our test framework can roll back database changes when all tests finish.

Reply:
Should we just delete truncate method then?